modules/ucfunnelBidAdapter.js

```diff
@@ -331,6 +331,11 @@ function addUserId(bidData, userId) {
           bidData[userIdProviderKey + '_linkType'] = userIdObjectOrValue.ext.linkType;
         }
         break;
+      case 'uid2':
+        if (userIdObjectOrValue.id) {
+          bidData['eids'] = userIdProviderKey + ',' + userIdObjectOrValue.id
```

Comment:
this is different from the other ids you're supporting above...is that intentional?

Reply:
yes